src/app/background/sagas/index.ts

```diff
@@ -24,7 +24,9 @@ import installationDetailsSaga from './installationDetails.saga';
 import { loginSaga } from './user.saga';
 
 const tracker =
-  process.env.TRACKING_SITE_ID && process.env.TRACKING_URL
+  process.env.TRACKING_SITE_ID &&
+  process.env.TRACKING_URL &&
+  navigator.doNotTrack === null
```

Comment:
Great to improve data privacy :) But I think it would be better to use `navigator.doNotTrack !== 1` because the user could have opted in for tracking.
Looking at the spec, it might be either 1 (no tracking), 0 (tracking accepted), null or undefined.

https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack

Reply:
Completely true! Hope I will have the opportunity to work with you ;)

Reply:
Typescript was complaining about `navigator.doNotTrack !== 1`. By reading the doc more carefully I just realized that I did'nt apply well the condition well for all navigators and all versions. The new commit should work. Maybe we could think of having an end-to-end test system to be sure that all changes can apply to all navigators or not.

Reply:
Indeed you're right, it's not `1` but rather `"1"` no?

Your code is really interesting and seems thorough, but as Dis-moi is only targetting Chrome and Firefox (am I wrong here?), I supppose adding a lot a code for IE is not worth the added complexity. I would rather try to keep things as simple as possible.

Reply:
Agreed

Reply:
I confirm the need : IE is not targeted. Just Firefox, Chrome and browsers based on those (Opera, Edge). Althought the latter  have a few users, so we don"t need to verify if it works with them.